### ISSUE TITLE
RangeError: Incorrect locale information provided (triggered via assistant_helper.localizeText)

### ISSUE THREAD

**Issue by tobias-klein** [ezra-bible-app/ezra-bible-app#471]
This has been automatically reported via Sentry.

A detailed Stacktrace is visible here:
https://sentry.io/share/issue/318d76e0d5234003a3320f93153b99d9/

We may simply need to catch this exception in assistant_helper.localizeText.

**Comment by zhuiks** [comment]
I think the problem lies in [`module_assistant/assistant_controller.js`](https://github.com/ezra-project/ezra-project/blob/bc64c33eecc9d3feb663225899de0420b203085b/app/frontend/components/module_assistant/assistant_controller.js#L45):
```javascript
  const lastUpdate = await ipcSettings.get('lastSwordRepoUpdate', null);
  const repositoriesAvailable = lastUpdate && await ipcNsi.repositoryConfigExisting();

  if (repositoriesAvailable) {
    state.reposUpdated = new Date(Date.parse(lastUpdate));
  }
```

`lastUpdate` can be anything...

**Comment by zhuiks** [comment]
Yes, based on stacktrace seems like `lng = <anonymous>`. This is strange because `lng` supplied by `i18n` itself
